api/README.md

```diff
@@ -6,7 +6,8 @@ You will need an environment file. See: `.env.example`. Contact current maintain
 
 - Automated (mac only):
```

Comment:
Could you add a note somewhere in this section reminding people to create the python alias in their bash profile? Some of us dolts will easily forget to do that.

Reply:
Please

Reply:
Done in: https://github.com/bcgov/wps/pull/2404/commits/c01c06ff92bb5ebe0e4a00764c771c03ae7496d9